command/job_init.go

```diff
@@ -48,6 +48,10 @@ func (c *JobInitCommand) AutocompleteFlags() complete.Flags {
 		})
 }
 
+func (c *JobInitCommand) AutocompleteArgs() complete.Predictor {
+	return complete.PredictNothing
```

Comment:
-short?

Reply:
This is already specified here: https://github.com/hashicorp/nomad/pull/4273/files/0bd7c0325c45fb6be80fce33c1176fecb0233a55#diff-2974fe73acd33761ac6948092972b347R47, but I thought `-short` doesn't take arguments, which is why this is `complete.PredictNothing`